ios/Swift/imagepicker/ImagePickerViewController.swift

```diff
@@ -64,78 +64,84 @@ extension ViewController {
+                        }
+                    } else {
+                        self.faceResults.text = "No faces found"
                     }
-                    // Get emotion likelihood as a % and display in UI
-                    for (emotion, total) in emotionTotals {
-                        let likelihood:Double = total / Double(numPeopleDetected)
-                        let percent: Int = Int(round(likelihood * 100))
-                        self.faceResults.text! += "\(emotion): \(percent)%\n"
-                    }
-                } else {
-                    self.faceResults.text = "No faces found"
-                }
-                
-                // Get label annotations
-                let labelAnnotations: JSON = responses["labelAnnotations"]
-                let numLabels: Int = labelAnnotations.count
-                var labels: Array<String> = []
-                if numLabels > 0 {
-                    var labelResultsText:String = "Labels found: "
-                    for index in 0..<numLabels {
-                        let label = labelAnnotations[index]["description"].stringValue
-                        labels.append(label)
-                    }
-                    for label in labels {
-                        // if it's not the last item add a comma
-                        if labels[labels.count - 1] != label {
-                            labelResultsText += "\(label), "
-                        } else {
-                            labelResultsText += "\(label)"
+                    
+                    // Get label annotations
+                    let labelAnnotations: JSON = responses["labelAnnotations"]
+                    let numLabels: Int = labelAnnotations.count
+                    var labels: Array<String> = []
+                    if numLabels > 0 {
+                        var labelResultsText:String = "Labels found: "
+                        for index in 0..<numLabels {
+                            let label = labelAnnotations[index]["description"].stringValue
+                            labels.append(label)
+                        }
+                        for label in labels {
+                            // if it's not the last item add a comma
+                            if labels[labels.count - 1] != label {
+                                labelResultsText += "\(label), "
+                            } else {
+                                labelResultsText += "\(label)"
+                            }
                         }
+                        self.labelResults.text = labelResultsText
+                    } else {
+                        self.labelResults.text = "No labels found"
                     }
-                    self.labelResults.text = labelResultsText
-                } else {
-                    self.labelResults.text = "No labels found"
                 }
+                
+            } catch {
```

Comment:
It sounds like this was why you submitted the PR, are there steps to reproduce the issue?